rest/route/version_routes.go

```diff
@@ -75,6 +81,81 @@ func (vh *versionHandler) Execute(ctx context.Context, sc data.Connector) (Respo
 	}, nil
 }
 
+// activeVersionsHandler is a RequestHandler for fetchhing
+// all active versions for given projectId
+type activeVersionsHandler struct {
+	projectId string
+}
+
+func getActiveVersionsRouteManager(route string, version int) *RouteManager {
+	return &RouteManager{
+		Route: route,
+		Methods: []MethodHandler{
+			{
+				Authenticator:  &NoAuthAuthenticator{},
+				RequestHandler: &activeVersionsHandler{},
+				MethodType:     http.MethodGet,
+			},
+		},
+		Version: version,
+	}
+}
+
+// Handler returns a pointer to a new activeVersionsHandler.
+func (avh *activeVersionsHandler) Handler() RequestHandler {
+	return &activeVersionsHandler{}
+}
+
+// ParseAndValidate fetches the versionId from the http request.
+func (avh *activeVersionsHandler) ParseAndValidate(
+	ctx context.Context,
+	r *http.Request,
+) error {
+	avh.projectId = getProjectIdFromRequest(r)
```

Comment:
Use MustHaveProjectCtx for this.

Reply:
MustHaveProjectContext, sorry.

Reply:
Cannot agree here. Actually, this was discussed with the team already - there was persons with opposite opinion on it. Anyway, in my case, this implementation is the only possible way, because I, most probably, will need cross-project requests (this is a functional requirement for perf. discovery feature).